src/scanpy/preprocessing/_pca/__init__.py

```diff
@@ -123,7 +123,7 @@ def pca(
             Not available with *dask* arrays.
         `'covariance_eigh'`
             Classic eigendecomposition of the covariance matrix, suited for tall-and-skinny matrices.
-            With dask, array must be CSR and chunked as (N, adata.shape[1]).
+            With dask, array must be CSR or dense and chunked as (N, adata.shape[1]).
```

Comment:
for `None` above it says

> For sparse *dask* arrays, will use `'covariance_eigh'`.

With this PR, what does `None` do for dense dask arrays?